.storybook/main.ts

```diff
@@ -3,7 +3,11 @@ import type { StorybookConfig } from '@storybook/nextjs';
 
 const config: StorybookConfig = {
   stories: ['../components/**/*.stories.tsx'],
-  addons: ['@storybook/addon-controls', '@storybook/addon-interactions'],
+  addons: [
+    '@storybook/addon-controls',
+    '@storybook/addon-interactions',
+    'storybook-addon-pseudo-states',
```

Comment:
How well maintained is this storybook-addon-pseudo-stories?

Reply:
This library is actively maintained, last update was 23 days ago

It has 147k weekly downloads on npm

Socket Report
![Screenshot_20230913_201445](https://github.com/nodejs/nodejs.org/assets/52033975/6d1e7219-3a87-4503-88dc-07ed4f6a4e2a)

Reply:
it's maintained by chromatique
https://github.com/chromaui/storybook-addon-pseudo-states